freedomofpress/securedrop
# docs: l10n: document the reviewer workflow

## Status

Ready for review

## Description of Changes

document the reviewer workflow

## Testing

* login as a reviewer (an admin is a reviewer by default)
* pick a string in a n incomplete language to not disrupt an existing language
* login as a non reviewer (create an account)
* verify the non reviewer interface changes depending on the state of the Approved button

## Deployment

N/A

## Checklist

### If you made changes to documentation:
